test/e2e/configuration_anomaly_detection_test.go

```diff
@@ -48,24 +68,204 @@ var _ = Describe("Configuration Anomaly Detection", Ordered, func() {
 			Expect(awsAccessKey).NotTo(BeEmpty(), "awsAccessKey not found")
 			Expect(awsSecretKey).NotTo(BeEmpty(), "awsSecretKey not found")
 
-			_, err := session.NewSession(aws.NewConfig().WithCredentials(credentials.NewStaticCredentials(awsAccessKey, awsSecretKey, "")).WithRegion(region))
+			_, err := session.NewSession(aws.NewConfig().WithCredentials(
+				credentials.NewStaticCredentials(awsAccessKey, awsSecretKey, "")).WithRegion(region))
 			Expect(err).NotTo(HaveOccurred(), "Could not set up aws session")
-			Expect(err).NotTo(HaveOccurred(), "Unable to get service logs for cluster")
-
-			// TODO(SDE-4821): Add the following tests
-			// AWS CCS: cluster has gone missing (no known misconfiguration)
-			// AWS CCS: cluster has gone missing (blocked egress)
-			// AWS CCS: cluster has gone missing (infra nodes turned off)
-			// AWS CCS: monitoring error budget burn (misconfigured UWM configmap)
-			// AWS CCS: monitoring errror budget burn (no known misconfiguration
-			ocmCli.Connection.Close()
+		}
+	})
+
+	It("AWS CCS: cluster has gone missing (blocked egress)", Label("aws", "ccs", "chgm", "limited-support", "blocking-egress"), func(ctx context.Context) {
+		if provider == "aws" {
+			awsAccessKey := os.Getenv("AWS_ACCESS_KEY_ID")
+			awsSecretKey := os.Getenv("AWS_SECRET_ACCESS_KEY")
+			Expect(awsAccessKey).NotTo(BeEmpty(), "AWS access key not found")
+			Expect(awsSecretKey).NotTo(BeEmpty(), "AWS secret key not found")
+
+			awsCli, err := awsinternal.NewClient(awsAccessKey, awsSecretKey, "", region)
+			Expect(err).NotTo(HaveOccurred(), "Failed to create AWS client")
+
+			clusterResource, err := ocme2eCli.ClustersMgmt().V1().Clusters().Cluster(clusterID).Get().Send()
+			Expect(err).NotTo(HaveOccurred(), "Failed to fetch cluster from OCM")
+
+			cluster := clusterResource.Body()
+			infraID := cluster.InfraID()
+			Expect(infraID).NotTo(BeEmpty(), "InfraID missing from cluster")
+
+			sgID, err := awsCli.GetSecurityGroupID(infraID)
+			Expect(err).NotTo(HaveOccurred(), "Failed to get security group ID")
+
+			ginkgo.GinkgoWriter.Printf("About to block egress for security group: %s\n", sgID)
+			Expect(awsinternal.BlockEgress(ctx, awsCli.Ec2Client, sgID)).To(Succeed(), "Failed to block egress")
+
+			err = PostStoppedInfraLimitedSupport(clusterID, ocmCli)
+			ginkgo.GinkgoWriter.Printf("Limited support reason posted. Restoring egress...\n")
+
+			lsResponse, err := GetLimitedSupportReasons(ocme2eCli, clusterID)
+			Expect(err).NotTo(HaveOccurred(), "Failed to get limited support reasons")
+
+			var reasonID string
+			lsReasons := lsResponse.Items()
+			for i := 0; i < lsReasons.Len(); i++ {
+				reason := lsReasons.Get(i)
+				if reason.Summary() == egressLS.Summary {
+					reasonID = reason.ID()
+					break
+				}
+			}
+
+			Expect(reasonID).NotTo(BeEmpty(), "Failed to find the posted limited support reason")
+			ginkgo.GinkgoWriter.Printf("Egress blocked\n")
+			time.Sleep(20 * time.Minute)
```

Comment:
The reason we wait 20 minutes is that this is the threshhold for the alert to trigger if the cluster no longer provides a heartbeat (which it can't, because of the blocked egress). 

Can you open a follow-up card to allow us to manually create the alert as part of our e2e toolset? This will allow us to:
1. misconfigure
2. create alert (manually vs waiting for it to trigger)
3. validate CAD's handling
4. revert misconfiguration